### PR TITLE
9119 - refactor isLoggedIn logic to check if token is set

### PR DESCRIPTION
**Pull request by codyseibert**
we think checking if the token is on the state is a better way to know if the user is logged in or not.

### CODE REVIEW

**Review by mrinsin — APPROVED** [review]
Awesome, thanks for doing this!

**Review by TomElliottFlexion — APPROVED** [review]

File: web-client/src/presenter/actions/isLoggedInAction.test.js [hunk 1]
```diff
@@ -21,23 +18,19 @@ describe('isLoggedInAction', () => {
   });
 
   it('should call path.isLoggedIn if currentUser is defined', async () => {
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/isLoggedInAction.test.js:20]: Nit: I think these test names need a small update

File: web-client/src/presenter/actions/isLoggedInAction.js [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
  * @returns {object} the list of section work items
  */
-export const isLoggedInAction = async ({
-  applicationContext,
-  path,
-  router,
-}) => {
-  const user = applicationContext.getCurrentUser();
-  if (user) {
+export const isLoggedInAction = async ({ get, path, router }) => {
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/isLoggedInAction.js:11]: Does this mean we can refactor `disengageAppMaintenanceSequence` to remove the call to isTokenOnStateAction?

Comment (codyseibert) [reply web-client/src/presenter/actions/isLoggedInAction.js:11]: yeah I think so, I'll remove it

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**